### ISSUE TITLE
Support Music IP Mix plugin

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#97]
```
I just downloaded squeezer, and so far it looks much better than the alternatives. I
was just wondering how hard it would be to support the 'music ip' mix plugin?  It is
an amazing addition to the server, and I rarely go without it. 

You can generate a mix by sending an HTTP request to HTTP://server/plugins/MusicMagic/musicmagic.HTML?song/artist/album=###&player=playerip

Requested by: davemiedema@gmail.com
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2014-08-03 15:44:46

**Comment by nikclayton** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer

**Comment by kaaholst** [comment]
Fixed in 2.0.0
